### ISSUE TITLE
Information Value whole dataset - Scorecard

### ISSUE THREAD

**Issue by billy-odera** [guillermo-navas-palencia/optbinning#51]
How do i compute the information value for the entire dataset?

Two, is it advisable to do the binning on train set and later use the bins for test set or bin the entire dataset ?

**Comment by guillermo-navas-palencia** [comment]
Hi,

You can compute the information value for all variables in the dataset using the BinningProcess class. After fitting, retrieve information for all variables using method ``summary()``. See tutorial: http://gnpalencia.org/optbinning/tutorials/tutorial_binning_process_telco_churn.html and API: http://gnpalencia.org/optbinning/binning_process.html.

Regarding your second question, this is a modelling decision, but it is advisable to split the dataset to reduce overfitting.

**Comment by billy-odera** [comment]
Can I manually adjust the bins?

**Comment by guillermo-navas-palencia** [comment]
Hi @billy-odera,

You can manually adjust bins using parameter ``user_splits`` and ``user_splits_fixed``. See an example here: http://gnpalencia.org/optbinning/tutorials/tutorial_binary.html#User-defined-split-points

If you have a particular example, or you can share your data, I can assist you if you encounter any difficulty.

**Comment by billy-odera** [comment]
Thanks a lot
